libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -243,6 +243,7 @@ const AP_Param::GroupInfo RC_Channel::var_info[] = {
     // @Values{Plane}: 179:ICEngine start / stop
     // @Values{Copter, Plane}: 180:Test autotuned gains after tune is complete
     // @Values{Plane}: 181: QuickTune
+    // @Values{Copter}: 182: In flight AHRS trim
```

Comment:
```suggestion
    // @Values{Copter}: 182: AHRS trim save
```
I think this is better,  "In flight AHRS trim save" would work also, but a bit long since MP cuts off all of these long ones in the dialog box

Reply:
I didn't want people to confuse it with a "set level" like you might do from the GCS on the bench. The build server description from the original PR is "Enable Copter in-flight AHRS Trim calibration" so this also fits with that.

Reply:
But I don't have any very strong feelings, just want to get it documented.

Reply:
This is most definitly not "save trim"!  We have a separate option which does an instantaneous save of your trim, and this is not that!

This could be `In flight calibration of AHRS Level Trim via RC stick input"

Reply:
Pete and I discussed and settled on a name...